glib/src/value.rs

```diff
@@ -454,6 +462,42 @@ impl Uninitialized for Value {
     }
 }
 
+impl ToValue for Value {
+    fn to_value(&self) -> Value {
+        self.clone()
+    }
+
+    fn value_type(&self) -> Type {
+        self.type_()
+    }
+}
+
+pub struct NopChecker;
```

Comment:
What is this good for and why is it public?

Reply:
It an unfortunate detail of the `FromValue` trait, see the lines that follow. It requires a `ValueTypeChecker` and it must be public. The existing `ValueTypeChecker` implementations didn't fit the current situation, so I had to write another one.

Reply:
Makes sense